mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -220,15 +218,30 @@ def process_data(num_rows=None, output_missing_csv_path=None):
     num_rows (int): Number of rows to process. If None, all rows are processed.
     output_missing_csv_path (str): If provided, movies that could not be matched will be written to a CSV at this path.
     """
+
+    if not DATA_DIR.exists():
+        raise FileNotFoundError(
+            f"Data directory does not exist at {DATA_DIR}, please create a new directory containing the netflix prize dataset files\n"
+            "https://archive.org/details/nf_prize_dataset.tar"
```

Comment:
We could extract the archive.org link into an environment variable, or just omit entirely depending on how we want to expose the dataset download?

Reply:
Not sure what you mean. Someone was working on automatic download of the Netflix data. I think @JamesKohlsRepo

Reply:
Since we reuse the link twice I was thinking about pulling it up somewhere but you're right, since James is working on it.